repos/fdbt-admin/src/pages/Reporting.tsx

```diff
@@ -0,0 +1,170 @@
+            const { client, userPoolId } = await getCognitoClientAndUserPool();
+
+            return listUsersInPool(client, userPoolId);
+        };
+
+        const getExportsFromMatchingDataBucket = async (): Promise<ObjectList> => {
+            const { client } = await getS3Client();
+            const bucketName = getBucketName(NETEX_DATA_BUCKET_PREFIX);
+            return listBucketObjects(client, bucketName);
+        };
+
+        getUsers()
+            .then((data) => {
+                const filteredData = data.filter((user) => {
+                    const email = user.Attributes?.find((attribute) => attribute.Name === 'email')?.Value as string;
+                    const hasTestNoc =
+                        (user.Attributes?.find((attribute) => attribute.Name === 'custom:noc')?.Value as string) ===
+                        'IWBusCo';
+                    const emailToLower = email.toLowerCase();
+                    const accountConfirmed = user?.UserStatus === 'CONFIRMED';
+
+                    if (
+                        !emailToLower.includes('kpmg') &&
+                        !emailToLower.includes('dft.gov.uk') &&
+                        !hasTestNoc &&
+                        accountConfirmed
+                    ) {
+                        return user;
+                    }
+
+                    return false;
+                });
+
+                setUsers(filteredData);
+
+                const nocs: string[] = [];
+
+                filteredData.forEach((user) => {
+                    const nocString = user.Attributes?.find((attribute) => attribute.Name === 'custom:noc')
+                        ?.Value as string;
+                    const usersNocs = [];
+                    if (nocString.includes(',')) {
+                        const splitNocs = nocString.split(',');
+                        splitNocs.forEach((noc) => {
+                            usersNocs.push(noc);
+                        });
+                    } else {
+                        usersNocs.push(nocString);
+                    }
+
+                    usersNocs.forEach((noc) => {
+                        if (!nocs.includes(noc)) {
+                            nocs.push(noc);
+                        }
+                    });
+                });
+
+                setRegisteredNocs(nocs);
+
+                getExportsFromMatchingDataBucket()
+                    .then((netexData) => setNetexFiles(netexData))
+                    .catch((err) => {
+                        console.error(err);
+                        setNetexFiles([]);
+                    });
+
+                setLoaded(true);
+            })
+            .catch((err) => {
+                console.error(err);
+
+                setUsers([]);
+            });
+    }, []);
+
+    const placeholder = (
+        <Table.Row>
+            <Table.Cell>Loading</Table.Cell>
+        </Table.Row>
+    );
+
+    const table = (
```

Comment:
could variables that hold elements be capital e.g. Table?

Reply:
we only use capital letters for components or types, this is just a variable.